guides/mix_tasks.md

```diff
@@ -422,9 +422,10 @@ And then `assets/` which should look similar to this:
 │   └── app.css
 ├── js
 │   └── app.js
-├── tailwind.config.js
+├── tailwind_heroicons.js
```

Comment:
We can discuss this after merging this PR but I wonder if this should go into `vendor/heroicons.js` for consistency with daisyui.js.

Reply:
moved